snowflake-cortex/README.md

```diff
@@ -0,0 +1,18 @@
+This Native App examplifies how to implement the Cortex Complete and to make it interact with user data.
+
+For this usecase, the dataset used is rather small, with only 10 entries, this is because the language model used in the Cortex function restricts the bytes of input data size and costs. YOu can change the language model used to a bigger or different style according to your necessities.
+For more information about it please visit **[this page](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#cost-considerations)**.
+
+To run this example first execute this command:
+```sh
+snow sql -f 'prepare/prepare_data.sql'
+```
+
+Then run `snow app run` on your terminal.
+
+To delete the database and the app run
+
+```sh
+snow sql -q 'DROP DATABASE SPOTIFY_CORTEX_DB;'
+snow app teardown
+```
```

Comment:
Are there any other docs you would like to add to the further reading section? Also, please take a look at the other README.mds and make sure the style is aligned.